client/src/global.scss

```
@@ -4,57 +4,57 @@
 
 // Global
 * {
-	padding: 0;
-	margin: 0;
-	box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
 }
 
 html {
-	font-size: 62.50%;
-	scroll-behavior: smooth;
+    font-size: 62.50%;
+    scroll-behavior: smooth;
 }
 
 body {
-	max-width: 600px;
-	margin: 0 auto;
+    max-width: 600px;
+    margin: 0 auto;
 
-	font-family: $ff-roboto;
+    font-family: $ff-roboto;
```

Da fällt mir ein: Wenn möglich noch eine Variable für die Text Farbe :)
```suggestion
    font-family: $ff-roboto;
    color: $fc-text
```

---

done!